src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -197,7 +197,7 @@ private function parseDefinition(\DOMElement $service, string $file, array $defa
         if ($alias = $service->getAttribute('alias')) {
             $this->validateAlias($service, $file);
 
-            $this->container->setAlias((string) $service->getAttribute('id'), $alias = new Alias($alias));
+            $this->container->setAlias($service->getAttribute('id'), $alias = new Alias($alias));
```

Comment:
`getAttribute()` returns a string (https://www.php.net/manual/en/domelement.getattribute.php)

I traced this cast back to https://github.com/symfony/symfony/commit/33c91f9be8e48fb085ab28019716eddc924c5904#diff-b56f1c0f8bef9890bdb59260a8ae980ad6c92185e30adb367678ce7056f373c0R127 where there kept by error (I think?)